src/model.ts

```diff
@@ -139,10 +140,10 @@ export interface IJobDefinitionModel extends ICreateJobModel {
 }
 // Convert an IDescribeJobModel to an IJobDetailModel
 export function convertDescribeJobtoJobDetail(
-  dj: Scheduler.IDescribeJob
+  jd: Scheduler.IDescribeJob
```

Comment:
`dj` stands for **D**escribe**J**ob; why change it to `jd`?

Reply:
It makes sense. Somehow couldn't make sense of what `dj` stands for, thought it's a typo for `JobDefinition`

Reply:
inside `convertDescribeJobtoJobDetail ` renamed `jd`/`dj` to `describeJob`